src/huggingface_hub/repocard.py

```diff
@@ -127,7 +128,9 @@ def save(self, filepath: Union[Path, str]):
         """
         filepath = Path(filepath)
         filepath.parent.mkdir(parents=True, exist_ok=True)
-        filepath.write_text(str(self), encoding="utf-8")
```

Comment:
Interesting, this didn't work as expected on Windows?

Reply:
It was "working" but encoding a newline was different on Windows (`"\n\r"`) than Linux/Macos (`"\n"`). That's what the `newline=""` is for otherwise you get a big diff if you edit a README file from different platforms. And `pathlib.Path.write_text` doesn't have this parameter in Python 3.7 so I had to use the standard `open(...)`.

(And yes, I discovered/learnt about all this while working on this PR :D)